Bangazon/Models/Product.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+//Written by:  Chaz Henricks
+namespace BangazonCLI
+{
+    // Gets and sets public properties of a Product.
+    public class Product
+    {
+        public int ProductId {get;}
+        public string Name {get; set;}
+        public string Description {get; set;}
+        public int Price {get; set;}
+        public DateTime? DateCreated {get; set;}
+        public Customer customer {get; set;}
```

Comment:
I think the question is are we going to need the whole customer object, or only the ID? I think the implementation will ultimately answer that question